locale/content.json

```diff
@@ -783,6 +784,14 @@
         "agency": "Department for Work and Pensions",
         "agencyAcronym": "DWP",
         "panel": "a judge"
+      },
+      "industrialInjuriesDisablement" : {
+        "acronym": "INDUSTRIAL INJURIES DISABLEMENT BEREAVEMENT BENEFIT",
```

Comment:
INDUSTRIAL INJURIES DISABLEMENT BENEFIT not INDUSTRIAL INJURIES DISABLEMENT **BEREAVEMENT** BENEFIT